roles/foreman_installer_devel_scenario/tasks/main.yml

```diff
@@ -20,6 +20,7 @@
   template:
     src: "{{ role_path }}/templates/{{ item }}"
     dest: /etc/foreman-installer/scenarios.d/{{ item }}
+    force: no
```

Comment:
@ehelms thinking about it more, I think I should also re-name this task to something like `Create initial devel scenario answers file` so that it's clear we won't overwrite the file once it exists

Reply:
I pushed an update which also renames this task so that the new behavior is more clear